### DISCUSSION TITLE
[Question]: Where did the View go where you Could See Bounding Box

### DISCUSSION THREAD

**Discussion by PablaV** [blakeblackshear/frigate#13221] [Ask A Question] [answered]
### What is your question:

In version <13.2 there was the ability to view the bounding box for a previous event and also see when when the object entered, left, became stationary in a zone. I can't seem to find that in 14.0. To be clear I am not talking about the debug view where you can real time see bounding boxes etc.

**Comment by NickM-27** [comment, marked as answer]
This is covered in the FAQ, it is not available in 0.14 directly. The snapshots can still be viewed on the frigate+ page and in the HA integration. 

A new pane to view this along with other information on the review page has already been added for the next version of frigate (0.15)
